gRPC.podspec

```diff
@@ -77,13 +84,12 @@ Pod::Spec.new do |s|
     gs.resource_bundles = { 'gRPC' => ['etc/roots.pem'] }
   end
 
+  # RPC library for ProtocolBuffers, based on gRPC
   s.subspec 'ProtoRPC' do |ps|
-    ps.summary  = 'RPC library for ProtocolBuffers, based on gRPC'
-
     ps.source_files = 'src/objective-c/ProtoRPC/*.{h,m}'
 
     ps.dependency 'gRPC/GRPCClient'
     ps.dependency 'gRPC/RxLibrary'
-    ps.dependency 'Protobuf', '~> 3.0'
+    ps.dependency 'Protobuf', '~> 3.0.0-alpha-3'
```

Comment:
I'm sending that on another PR, as I'm also simplifying things for Cocoapods users by invoking protoc in the podspec, as part of `pod install` (so it needs rewording).